pkg/cmd/target/drop_test.go

```diff
@@ -4,12 +4,11 @@ SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener con
 SPDX-License-Identifier: Apache-2.0
 */
 
-package drop_test
```

Comment:
why? leave it in the `drop_test` package and import cmd target as 
`cmdtarget "github.com/gardener/gardenctl-v2/pkg/cmd/target"`
same applies for the other test package renamings

Reply:
sure, we could do so. But at least for the targe_test file, we nee dto have it in the same packe or alternatively we nee to export `validTargetArgsFunction`
So I think wee need to agree on one approach here. Either we always include tests in the same package or we don't but accept that we need to export all things we'd like to test then. WDYT?

Reply:
added whitebox test file as discussed